suite-native/graph/src/components/Graph.tsx

```diff
@@ -1,26 +1,63 @@
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { LineGraph, GraphPoint } from 'react-native-graph';
 
 import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
-import { Box } from '@suite-native/atoms';
+import { Box, Text } from '@suite-native/atoms';
+import { sumLineGraphPoints } from '@suite-common/wallet-graph';
+
+import { AxisLabel } from './AxisLabel';
+import {
+    getAxisLabelPercentagePosition,
+    maxLineGraphPointArrayItemIndex,
+    minLineGraphPointArrayItemIndex,
+} from '../utils';
 
 type GraphProps = {
     points: GraphPoint[];
 };
 
 const graphWrapperStyle = prepareNativeStyle(_ => ({
     width: '100%',
-    height: 150,
-    marginVertical: 40,
+    height: 250,
+    marginVertical: 20,
+    justifyContent: 'center',
+    alignItems: 'center',
 }));
 
 const graphStyle = prepareNativeStyle(_ => ({
     alignSelf: 'center',
-    aspectRatio: 21 / 9,
+    aspectRatio: 1.5,
 }));
 
 export const Graph = ({ points = [] }: GraphProps) => {
     const { applyStyle } = useNativeStyles();
+    const [nonZeroBalance, setNonZeroBalance] = useState(false);
+
+    useEffect(() => {
+        setNonZeroBalance(sumLineGraphPoints(points) > 0);
+    }, [points]);
+
+    const extremaFromGraphPoints = useMemo(() => {
```

Comment:
I move this to utils file and write test for it

Reply:
It is moved